### ISSUE TITLE
v2x_libdem_covid not included in vdm dataset

### ISSUE THREAD

**Issue by turbanisch** [vdeminstitute/vdemdata#36]
I just installed the package from this repo but the `vdem` dataset does not seem to include the variable `v2x_libdem_covid`. I can access the codebook entry via `vdemdata::var_info` but the following yields an empty dataframe: 
```r
library(tidyverse)
vdemdata::vdem %>% select(matches("covid"))
```
I am a first-time Vdem-user so it might well be the case that I am missing something here.

**Comment by vdeminstitute** [comment]
Thanks for using our package, v2x_libdem_covid was created for internal use only and is not included in v11 or v11.1. If you are interested in our pandem data on violations of democratic standards during the Covid-19 pandemic, check it out here: https://github.com/vdeminstitute/pandem

**Comment by turbanisch** [comment]
Thank you so much!
